libevm/VMCalls.cpp

```diff
@@ -153,11 +153,20 @@ void VM::caseCall()
 	bytesRef output;
 	if (caseCallSetup(callParams.get(), output))
 	{
-		std::pair<bool, owning_bytes_ref> callResult = m_ext->call(*callParams);
-		callResult.second.copyTo(output);
-		m_returnData.assign(callResult.second.begin(), callResult.second.end());
-
-		m_SPP[0] = callResult.first ? 1 : 0;
+		bool success = false;
+		owning_bytes_ref outputRef;
+		std::tie(success, outputRef) = m_ext->call(*callParams);
+		outputRef.copyTo(output);
```

Comment:
Am I mistaken that `output` is only written to but not read from?  (This didn't change in this commit, but I started wondering now.)